common/src/main/java/org/keycloak/common/util/Time.java

```diff
@@ -69,6 +69,15 @@ public static long toMillis(long time) {
         return time * 1000L;
     }
 
+    /**
+     *  Calculates number of milliseconds from passed argument
+     * @param days Time in days
+     * @return Time in milliseconds
+     */
+    public static long daysToMillis(int days) {
+        return (long) days * 24 * 60 * 60 * 1000;
```

Comment:
I've just noticed that a small improvement might be made for this. I'll leave the rest of the review up to @keycloak/core team.

```suggestion
        return Duration.ofDays(days).toMillis();
```

Reply:
Nice, I think I'll just remove the method altogether.